features/visual-viewport.yml

```diff
@@ -1,17 +1,18 @@
 name: Visual viewport API
 description: The `visualViewport` API provides a way to query and modify the user-visible viewport of a web page.
 spec: https://drafts.csswg.org/cssom-view-1/#visualViewport
-# The resize and scroll events are excluded as the onresize and onscroll
-# properties were missing in Chrome 61, but were added in Chrome 62.
```

Comment:
I don't really understand this old comment. What's making us change our mind and now include these 2 keys?